src/components/UserBox/index.js

```diff
@@ -1,21 +1,54 @@
 import React, { Component } from 'react';
 import './user-box.scss'; 
-import UserPhoto from "../UserPhoto";
-import UserName from "../UserName";
-import UserComment from "../UserComment";
+import UserPhoto from '../UserPhoto';
+import UserName from '../UserName';
+import UserComment from '../UserComment';
+import PostDate from '../PostDate';
 
 export default class UserBox extends Component {
 
-  render() {
-    const { name, comment, photo } = this.props;
-    return (
-      <div className='user-box'>
-        <UserPhoto photo={ photo } />
-        <div className='text-box'>
-          <UserName name={ name } />
-          <UserComment comment={ comment } />
-        </div>
-      </div>
-    );
-  }
+    render() {
+        const { name, comment, photo, date } = this.props;
+        return (
+            <article className="slds-card slds-m-bottom--x-large">
+                <div className="slds-card__header slds-grid">
+                    <header className="slds-media slds-media--center slds-has-flexi-truncate">
+                        <div className="slds-media__figure">
+                    
+                        </div>
+                        <div className="slds-media__body">
+                    
+                        </div>
+                    </header>
+                    <div className="slds-no-flex">
+                  
+                    </div>
+                </div>
+                <div className="slds-card__body">
+                    <article className="slds-post">
+                      <header className="slds-post__header slds-media">
+                        <div className="slds-media__figure">
```

Comment:
por que en algunos lugares usa indentation 2 space y en otro 4 .. deberia ser 4 siempre ... be careful con eso dude